### DISCUSSION TITLE
How to log two cars in Teslamate?

### DISCUSSION THREAD

**Discussion by buzzhd** [teslamate-org/teslamate#2652] [Q&A]
Signed in with one car already? Is there a way to sign in second car so teslamate could log both info at one instance? Thanks!

**Comment by cwanja** [comment]
You’d have to have that second car under the account that’s already logged in. 

Share the second car with the current account. https://www.tesla.com/support/account-support

**Reply by buzzhd** [reply]
Thanks, will give it a try.
